docs/source/api/python/dgl.sparse_v0.rst

```diff
@@ -13,7 +13,7 @@ Sparse matrix class
 .. currentmodule:: dgl.mock_sparse
 
 .. autoclass:: SparseMatrix
-	:members: shape, nnz, dtype, device, row, col, val, __call__, indices, coo, csr, csc, dense
+	:members: shape, nnz, dtype, device, row, col, val, __call__, indices, coo, csr, csc, dense, T, transpose
```

Comment:
If we want to support both `t`, and `T`, then perhaps you also need to include `t` here, though I'm not sure if we want to do that.

Reply:
Sure. Will add it for now.

Reply:
done